### ISSUE TITLE
erlang rpm GPG check via Ansible Collection

### ISSUE THREAD

**Issue by dustin1530** [jfrog/JFrog-Cloud-Installers#289]
**Is this a request for help?**: No. 

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): Bug Report

Noticed in the Ansible collection, the [install/upgrade for RabbitMQ](https://github.com/jfrog/JFrog-Cloud-Installers/blob/master/Ansible/ansible_collections/jfrog/platform/roles/xray/tasks/rabbitmq/setup/RedHat.yml#L54) we are importing the RabbitMQ signing key, for RedHat installs. This is incorrect for the erlang rpm that is being installed as the next step. This signing key is not used to sign the erlang rpm. The task to install do work as we are disabling the GPG check for the erlang package install. In my environment I am unable to have the GPG check disabled, which is causing failures. I added the following task to import the erlang signing key and then flipped the disable_gpg_check to false. 

```yaml
- name: Import Erlang signing key
  become: true
  ignore_errors: true
  ansible.builtin.rpm_key:
    key: 'https://dl.cloudsmith.io/public/rabbitmq/rabbitmq-erlang/gpg.E495BB49CC4BBE5B.key'
    state: present
  until: import_rabbitmq_signing_key_result is succeeded
  delay: 5
  retries: 3

- name: Install erlang package
  become: true
  ansible.builtin.yum:
    name: "{{ xray_erlang_package }}"
    state: present
    disable_gpg_check: false
  vars:
    ansible_python_interpreter: "{{ yum_python_interpreter }}"
  register: install_erlang_package_result
  when: check_erlang_package_result.matched > 0
``` 

RPM and Singing key information:
Checking RPM package signature information, note the `Key ID`
```bash
# rpm --checksig --verbose erlang-25.0.3-1.el8.x86_64.rpm
erlang-25.0.3-1.el8.x86_64.rpm:
    Header V4 RSA/SHA256 Signature, key ID cc4bbe5b: NOKEY
    Header SHA256 digest: OK
    Header SHA1 digest: OK
    Payload SHA256 digest: OK
    V4 RSA/SHA256 Signature, key ID cc4bbe5b: NOKEY
    MD5 digest: OK
```

Importing key, verifying key was imported and showing info about imported key:
```bash
# rpm --import rabbitmq-release-signing-key.asc

# rpm -q gpg-pubkey --qf '%{NAME}-%{VERSION}-%{RELEASE}\t%{SUMMARY}\n' | grep rabbitmq
gpg-pubkey-6026dfca-573adfde    gpg(RabbitMQ Release Signing Key <info@rabbitmq.com>)

# rpm -qi gpg-pubkey-6026dfca-573adfde
Name        : gpg-pubkey
Version     : 6026dfca
Release     : 573adfde
Architecture: (none)
Install Date: Thu 11 May 2023 01:18:08 PM CDT
Group       : Public Keys
Size        : 0
License     : pubkey
Signature   : (none)
Source RPM  : (none)
Build Date  : Tue 17 May 2016 04:09:50 AM CDT
Build Host  : localhost
Relocations : (not relocatable)
Packager    : RabbitMQ Release Signing Key <info@rabbitmq.com>
Summary     : gpg(RabbitMQ Release Signing Key <info@rabbitmq.com>)
Description :
-----BEGIN PGP PUBLIC KEY BLOCK-----
Version: rpm-4.14.3 (NSS-3)
<removed_hash_to_save_space>
```
The `key ID` from the `rpm --checksig` command does not match the `name` of the imported key via the `rpm -qi` command. 

However, the `key ID` and `name` do match when using the key from "https://dl.cloudsmith.io/public/rabbitmq/rabbitmq-erlang/gpg.E495BB49CC4BBE5B.key"
```bash
# rpm --import gpg.E495BB49CC4BBE5B.key

# rpm -q gpg-pubkey --qf '%{NAME}-%{VERSION}-%{RELEASE}\t%{SUMMARY}\n' | grep erlang
gpg-pubkey-cc4bbe5b-60490155    gpg(Cloudsmith Package (rabbitmq/rabbitmq-erlang) <support@cloudsmith.io>)

# rpm -qi gpg-pubkey-cc4bbe5b-60490155
Name        : gpg-pubkey
Version     : cc4bbe5b
Release     : 60490155
Architecture: (none)
Install Date: Thu 11 May 2023 01:22:40 PM CDT
Group       : Public Keys
Size        : 0
License     : pubkey
Signature   : (none)
Source RPM  : (none)
Build Date  : Wed 10 Mar 2021 11:26:45 AM CST
Build Host  : localhost
Relocations : (not relocatable)
Packager    : Cloudsmith Package (rabbitmq/rabbitmq-erlang) <support@cloudsmith.io>
Summary     : gpg(Cloudsmith Package (rabbitmq/rabbitmq-erlang) <support@cloudsmith.io>)
Description :
-----BEGIN PGP PUBLIC KEY BLOCK-----
<removed_hash_to_save_space>
```

Now installing with disabled_gpg_check set to false works as intended. 

**Which installer**:
Ansible collection

**Which product and version** (eg: ansible & collection version - 7.24.2):
Xray 

**Which operating system and version**(eg: ubuntu & version - 20.4):
RHEL 8 

**Which product license** (Enterprise/Pro):
enterprise

**JFrog support reference** (if already raised with support team): 

**What happened**:

**What you expected to happen**:

**How to reproduce it** (as minimally and precisely as possible):

**Anything else we need to know**:

**Comment by srinivasgowda097** [comment]
@dustin1530 Thanks for raising the issue, We have already identified and fixed internally, will be available in next minor 10.13.0 release

**Comment by chukka** [comment]
@dustin1530 looks like , It's already fixed in ansible platform collection `10.2.2`, please upgrade and try
